.github/workflows/main.yml

```diff
@@ -3,19 +3,36 @@ name: Main
 on: [push, pull_request]
 
 jobs:
+
   lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Setup Python
-        uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
         with:
           python-version: 3.8
-      - name: Install tox
-        run: python -m pip install tox
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools wheel tox
       - name: Run linting
         run: python -m tox -e lint
 
+  types:
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
```

Comment:
This duplicates the `test` matrix. It'd be nice to specify it once, as well as the "latest Python" to use in the other jobs.

Reply:
TIL about YAML anchors and aliases, which are [supported by Travis](https://docs.travis-ci.com/user/build-config-yaml), but [not (yet) by GitHub Actions](https://github.community/t/support-for-yaml-anchors/16128).

There's a discussion thread about [how to share `matrix` between jobs](https://github.community/t/how-to-share-matrix-between-jobs/128595/), but the [documented configuration](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#example-6) seems more complex than it's worth.